cmake/Modules/Findopm-parser.cmake

```diff
@@ -47,6 +47,9 @@ endif ()
 find_package(opm-parser CONFIG HINTS ${_opm_parser_build})
 if (opm-parser_FOUND)
     find_package(ecl REQUIRED)
+    find_package(Boost 1.44.0
+                 COMPONENTS regex filesystem system date_time
```

Comment:
Well - I agree in that; but there is still *some failure* with Boost test in downstream modules; it might be that the `FindBoost` induced by opm-parser in this PR conflicts with the `FindBoost( unit_test )` which comes from the `opm-output-prereqs.cmake` file?